### DISCUSSION TITLE
New Navigation Kit Questions

### DISCUSSION THREAD

**Discussion by amyodea** [twilio-labs/paste#3958] [General] [answered]
Hey Team! In the middle of implementation of the new UI kit for Console and have two questions. 

1. The empty state for the 'No product are pinned' in the left navigation stays the same light version when the team use the new kit. Are we okay with this dark version I designed on the right? 
![Screenshot 2024-06-24 at 10 01 56](https://github.com/twilio-labs/paste/assets/104831341/86a32036-0fe9-4232-9ffe-5058506661b7)

2. For the account switcher dropdown in the top navigation bar, the badge that displays the 'account name' and chevron and allows the customer to switch accounts. Currently, only the chevron is clickable, we want to make the entire badge clickable in this case so there are no usability issues, are we okay with the team customizing this for this instance of the badge in the top bar? 

Thank you in advance team!

**Comment by serifluous** [comment, marked as answer]
Hey @amyodea ! 

1. I think that empty state might be a Console-specific implementation, so you're welcome to use what's best for Console. I'd recommend using the inverse Anchor and checking that the color contrast is high enough with the text + background. 
2. For the Account Switcher, the reason the entire Badge isn't clickable is because the Account Switcher Button (the chevron) triggers a Menu, which means it shouldn't change labels. The way we've solved that is to make the account name just text, separate from the actual Account Switcher Button. If you set the entire Badge as clickable, you'll essentially be wrapping a button inside another button.

The current solution isn't perfect, but did you get any customer feedback or usage data on its usability that could help us iterate?

**Comment by amyodea** [comment]
1. Yep was thinking that too, updated, thanks! 
2. Okay yeah, I don't have customer feedback yet but will start tracking it now it's live. The chevron is a small clickable area so I anticipate it being a usability issue. Will report back! 

Thanks Sarah!
